src/test/java/elizCurtnazar/tests/tema7/HomeButton.java

```diff
@@ -0,0 +1,44 @@
+package elizCurtnazar.tests.tema7;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeButton extends BaseTest {
+
+    @Test
+    public void goToTheHomePage() {
+        WebElement menu = driver.findElement(By.cssSelector(".btn-dark.btn-lg.toggle"));
+        menu.click();
+        WebElement homeButton = driver.findElement(By.xpath("/html/body/nav/ul/li[2]/a"));
+        homeButton.click();
+        Assert.assertEquals(driver.getCurrentUrl(), "https://katalon-demo-cura.herokuapp.com/");
+
+    }
+
+    @Test
+    public void loginPage() {
+        driver.get("https://katalon-demo-cura.herokuapp.com/profile.php#login");
+        goToTheHomePage();
+
+    }
+
+    @Test
+    public void historyPage() {
+        driver.get("https://katalon-demo-cura.herokuapp.com/history.php#history");
+        goToTheHomePage();
+    }
```

Comment:
Please add an assertion at the end of the test case. Thanks!

Reply:
done